hyperSpec/R/FauxCell.R

```diff
@@ -0,0 +1,41 @@
+#'
+#' Faux Cell
+#'
+#' Faux Cell is a synthetic data set intended for testing and demonstration.
+#' It is small so that it processes quickly.
+#' There are 300 Raman-like spectra allocated to three groups/regions.
+#' The spectrum of each region is unique and simple, with a single peak with a particular
+#' frequency and line width.
+#' Each spectrum has 1400 data points.  A small amount of noise has been added.
+#' The three regions represent the sample matrix, a cell, and a nucleus within the cell.
+#' The data is indexed along x and y dimensions, simulating data collected on a grid.
```

Comment:
I like this detailed description. It seems to be similar to the chondro dataset, right?